File: week02/day_3/printer.py
__author__ = 'ylwoi'

# - Create a function called `printer`
#   which prints the input parameters
#   (can have multiple number of arguments)

def printer(*args):
    for i in range(0, len(args)):
        print(args[i], end=" ")

printer(input("Enter words: "))

File: week07/day3/printer.js
/**
 * Created by cw on 2017-05-03.
 */
'use strict';
// - Create a function called `printer`
//   which logs to the console the input parameters
//   (can have multiple number of arguments)

function printer () {
    Object.values(arguments).forEach(function(e){
        console.log(e);
    });
}

printer('hello', 'bello', 4);